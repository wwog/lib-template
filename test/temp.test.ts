import { assert, describe, expect, it } from 'vitest'
import { add, divide, multiply, subtract } from '../src'

describe('your test ', () => {
  it('add', () => {
    assert.equal(add(1, 1), 2)
  })

  it('subtract', () => {
    expect(subtract(4, 3)).eq(1)
  })

  it('snapshot', () => {
    expect({ foo: 'bar' }).toMatchSnapshot()
  })
})
