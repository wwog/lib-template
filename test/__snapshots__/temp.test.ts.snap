// Vitest Snapshot v1

exports[`your test  > snapshot 1`] = `
{
  "foo": "bar",
}
`;
